docs/sources/administration/data-source-management/team_lbac.md

```diff
@@ -0,0 +1,156 @@
+---
+aliases:
+  - ../datasources/datasource_permissions/team_lbac
+  - ../datasources/datasource_permissions/teamlbac
+  - ../enterprise/datasource_permissions/team_lbac
+  - ../enterprise/datasource_permissions/teamlbac
+  - ../permissions/datasource_permissions/team_lbac
+  - ../permissions/datasource_permissions/teamlbac
```

Comment:
Since this is a new file, do we need all of these aliases?

Aliases are for redirecting old page locations to this current page. Since the page is new, I don't think we need any redirects.

Reply:
removing all of the aliases

Reply:
thought they would be used as a "aliases" as they can use these different ways to get search results

Reply:
Ah yeah, that's a reasonable idea based upon the name of the front matter field, but it turns out to be only for redirects (https://grafana.com/docs/writers-toolkit/write/front-matter/#aliases).